src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,70 @@
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendarGap;
+
+  padding: $calendarPadding;
+  width: $daySquareSize * 7 + $calendarGap * 6;
+
+  font-size: 30px;
+  font-family: Arial , sans-serif;
+
+  @for $day from 1 through 31 {
+    &__day-#{$day} {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+
+      box-sizing: border-box;
+      width: $daySquareSize;
+      height: $daySquareSize;
+
+      background: #eee;
+      border: 1px solid #000;
+      cursor: pointer;
+      transition-duration: 350ms , 700ms ;
+      transition-property: transform, background-color ;
+
+      &:hover {
+        transition: transform 700ms;
+        transform: translate(10px , 15px);
```

Comment:
Was an optional task so is not wrong if is other animation

Reply:
@IoghiStudio I like your hover but now it looks a bit counterintuitive (me as user expect that it will go up on hover but not down). So I would recommend you to change the direction of your translate